src/api/app/models/package.rb

```diff
@@ -1368,6 +1369,12 @@ def convert_to_symsync
     BackendPackage.where(package_id: id).delete_all
     decline_requests_with_self_as_target("The package '#{project.name} / #{name}' is now maintained at #{scmsync}")
   end
+
+  def report_bug_url_is_not_local
+    return true unless report_bug_url
```

Comment:
Isn't this one an existence validation?

Reply:
It is not.

Reply:
Well no, this is a guard clause, if not, the line below crashes